tests/test_utils.py

```diff
@@ -46,3 +55,17 @@ def get_qasm_as_circuit(qasm_file: QasmFile) -> "Circuit":
     """
     this_dir = Path(Path(__file__).resolve()).parent
     return circuit_from_qasm(f"{this_dir}/data/qasm/{qasm_file.name}.qasm")
+
+
+def get_phir_json(qasmfile: QasmFile, rebase: bool) -> dict[str, Any]:  # noqa: FBT001
```

Comment:
```suggestion
def get_phir_json(qasmfile: QasmFile, *, rebase: bool) -> dict[str, Any]:
```
I like the last solution mentioned at https://docs.astral.sh/ruff/rules/boolean-type-hint-positional-argument/